src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String login;
```

Comment:
check common mistakes file, make email (login) unique

Reply:
Thank you. Added an annotation to make the column unique, but then removed it because I thought checking in the method registry would be enough.

Reply:
double check is a common practice :)